src/gausshermite.jl

```diff
@@ -268,7 +268,7 @@ function hermite_initialguess(n::Integer)
     T(t) = t^(2/3)*(1+5/48*t^(-2)-5/36*t^(-4)+(77125/82944)*t^(-6) -108056875/6967296*t^(-8)+162375596875/334430208*t^(-10))
     airyrts = -T.(3π/8*(4*(1:m) .- 1))
 
-    airyrts[1:11] = AIRY_ROOTS  # correct first 11.
+    airyrts[1:10] .= AIRY_ROOTS[1:10]  # correct first 10.
```

Comment:
@ajt60gaibb do you know if this is ok?

Reply:
In `v0.4.7`, there were duplicated hard-coded roots of Airy function.

* https://github.com/JuliaApproximation/FastGaussQuadrature.jl/blob/v0.4.7/src/constants.jl#L98
* https://github.com/JuliaApproximation/FastGaussQuadrature.jl/blob/v0.4.7/src/gausshermite.jl#L274

In PR #97, I was trying to resolve this duplication, but that was incorrect.
With the change in this line 271, the problem will be fixed.